static/app/views/settings/organizationDeveloperSettings/index.spec.tsx

```diff
@@ -159,6 +178,25 @@ describe('Organization Developer Settings', function () {
         })
       );
     });
+    it('cannot make a request to publish an integration', async () => {
+      MockApiClient.clearMockResponses();
+      MockApiClient.addMockResponse({
+        url: `/organizations/${org.slug}/sentry-apps/`,
+        body: [sentryApp],
+      });
+
+      const router = RouterFixture({
+        location: LocationFixture({query: {type: 'public'}}),
+      });
+
+      render(<OrganizationDeveloperSettings />, {
+        router,
+      });
+
+      const publishButton = await screen.findByRole('button', {name: 'Publish'});
+
+      expect(publishButton).toHaveAttribute('aria-disabled', 'true');
```

Comment:
It might be good to also check with `.toBeDisabled()` here to test functionality if possible. It looks like `.toHaveAttribute('aria-disabled', 'true')` is more so for accessibility